alice-ide/src/main/resources/org/alice/stageide/themes/FlatLaf.properties

```diff
@@ -0,0 +1,67 @@
+#todo, turn off maven checkstyle on this file.
+###################################################################
+#General UI color choices/overrides
+###################################################################
+#@foreground =
+#@background =
+
+# this controls the ui highlight in dropdowns n'at
+[light]@accentColor = rgb(16, 113, 158)
+[dark]@accentColor= rgb(26, 113, 103)
+
+@accentFocusColor = saturate(@accentColor, 25%)
+@buttonArrowColor = @accentColor
+@buttonHoverArrowColor = @accentFocusColor
+Component.arrowType = triangle
+
+###################################################################
+# Alice specific variables
+###################################################################
+Alice.Comment.Color = #ddd
+Alice.Comment.Color.foreground = #060
+
+# that yellow color for objects/classes
+[light]Alice.Noun.Color = #FDF6C0
+[dark]Alice.Noun.Color =  rgb(211, 174, 57)
+
+# types, currently the same as nouns. What is a type? what should this be?
+[light]Alice.Type.Color = $Alice.Noun.Color
+[dark]Alice.Type.Color =  $Alice.Noun.Color
+Alice.Type.Color.muted = desaturate($Alice.Type.Color, 10%)
+
+# logically, Constructor color should relate to nouns, so making that more explicit
+[light]Alice.Constructor.Color = spin($Alice.Noun.Color, -10)
+[dark]Alice.Constructor.Color = spin($Alice.Noun.Color, -10)
+Alice.Constructor.Block.Color = $Alice.Constructor.Color
+
+# fields/properties (just the gallery models initiated?) (orange sherbert)
+[light]Alice.Field.Color = #D6AC8B
+[dark]Alice.Field.Color = rgb(200, 120, 109)
+
+# procedures are purple/which in the mass of purple reads pretty neutral. should this be purple or neutral?
+[light]Alice.Procedure.Color = #B2B7D9
+[dark]Alice.Procedure.Color = rgb(81, 165, 219)
+Alice.Procedure.Block.Color = #d3d7f0
+
+# functions are green
+[light]Alice.Function.Color = #ADCF95
+[dark]Alice.Function.Color =rgb(150, 190, 113)
+Alice.Function.Block.Color = #d3e7c7
+
+# Events, a mildly different purple?
+[light]Alice.Event.Color = #d3d7f0
+[dark]Alice.Event.Color = rgb( 143, 77, 107)
+
+# resources need a color too, randomly picked a purple cause we like purple
+[light]Alice.Resource.Color = #fff
```

Comment:
Wasn't this ffffff in the old DefaultTheme?

I am not a fan of this mix of ways to express colors, but I can only complain because you brought them together to be seen.